src/mozillavpn.cpp

```diff
@@ -1381,6 +1381,11 @@ void MozillaVPN::registerUrlOpenerLabels() {
   uo->registerUrlLabel("upgradeToAnnualUrl", []() -> QString {
     return Constants::upgradeToAnnualUrl();
   });
+
+  uo->registerUrlLabel("sumoPrivacy", []() -> QString {
+    return "https://support.mozilla.org/en-US/kb/"
```

Comment:
Same comment here, let's remove "en-US" so that moz.org can determine and reroute to the correct locale.